csharp/Platform.IO/ConsoleHelpers.cs

```diff
@@ -38,9 +82,29 @@ public static string GetOrReadArgument(int index, string readMessage, params str
             }
         }
 
+        /// <summary>
+        /// <para>Output a input string to the console. The method is only executed if the application has been compiled with the DEBUG directive.</para>
+        /// <para>Выводит переданную строку в консоль. Метод выполняется только в том случае, если приложение было скомпилировано с директивой DEBUG.</para>
+        /// </summary>
+        /// <param name="string">
+        /// <para>The string, that you want to output to the console.</para>
+        /// <para>Строка, которую нужно вывести в консоль.</para>
+        /// </param>
         [Conditional("DEBUG")]
         public static void Debug(string @string) => Console.WriteLine(@string);
 
+        /// <summary>
+        ///	<para>Output composit formatted text in the console.</para>
+        /// <para>Выводит строку составного формата в консоли.</para>
+        /// </summary>
+        /// <param name="format">
+        /// <para>A composite format string.</para>
+        /// <para>Строка составного формата.</para>
+        /// </param>
+        /// <param name="args">
+        /// <para>An array of objects to write using format.</para>
+        /// <para>Массив объектов для записи с использованием format.</para>
```

Comment:
In English version of the documentation use can click "Edit" button, which will show you the XML source of the documentation. And you can find [there](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/Console.xml#L4527) that link to the parameter name is used like so `<paramref name="format" />`. Link to `format` parameter should be applied both for English and Russian version.